dialplan/asterisk/extensions_lib_user.conf

```diff
@@ -168,6 +170,11 @@ same  =             n,Hangup()
 same  =             n(busy),Playback(busy-call)
 same  =             n,Hangup()
 
+exten = blocked,1,NoOp(Blocked call from caller id number ${CALLERID(num)})
+same  =         n,CELGenUserEvent(WAZO_USER_BLOCKED_CALL,blocked_caller_id: ${CALLERID(all)}, destination_user_uuid: ${WAZO_DST_UUID}, blocked_number_uuid: ${WAZO_BLOCKED_NUMBER_UUID})
```

Comment:
Please split `CALLERID(all)` into name and num to avoid parsing the callerid in wazo-call-logd